// C-ERR

func main() {
    {
        var a = 1
        printnum(a)
    }

    printnum(a)
}